### PR TITLE
#1569: Resumes.resume(login) implementation.

### PR DESCRIPTION
**Pull request by paulodamaso**
For #1569: Resumes.resume(login) implementation.
- Added `Resume` interface modeling a resume, with `Fake` implementation for tests
- Added `Resumes.resume(login)` method in resume
- Added test for `Resumes.resume(login)` in `ResumesTest`
- Left puzzle for `Resumes.resume(login)`

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#1631](https://github.com/zerocracy/farm/issues/1631) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2340645151); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/zerocracy/pmo/ResumesTest.java [hunk 1]
```diff
@@ -101,4 +102,40 @@ public void findUnassignedResumes() throws Exception {
             Matchers.contains(first)
         );
     }
+
+    @Test(expected = UnsupportedOperationException.class)
+    public void findResume() throws Exception {
+        final Farm farm = new FkFarm();
+        final String login = "login";
+        final LocalDateTime time = LocalDateTime.of(
```
Comment (krzyk) [inline src/test/java/com/zerocracy/pmo/ResumesTest.java]: @paulodamaso use `Instant` and `Instant.parse("2018-01-01T00:00:00Z")`

**Comment by paulodamaso** [comment]
@krzyk Made the corrections, please take a look.

File: src/test/java/com/zerocracy/pmo/ResumesTest.java [hunk 2]
```diff
@@ -126,7 +125,7 @@ public void findResume() throws Exception {
         final Resumes resumes = new Resumes(farm).bootstrap();
         resumes.add(
             login,
-            time,
+            LocalDateTime.ofInstant(time, ZoneId.of("UTC")),
```
Comment (krzyk) [inline src/test/java/com/zerocracy/pmo/ResumesTest.java]: @paulodamaso `ZoneId.of("UTC")` -> `ZoneOffset.UTC`

**Comment by paulodamaso** [comment]
@krzyk Done, please take a look.

**Comment by krzyk** [comment]
@paulodamaso thanks

**Comment by krzyk** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@g4s8 please take a look, this task is delaying #1568

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 38 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1631](https://github.com/zerocracy/farm/issues/1631) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +25 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**